README.adoc

```diff
@@ -22,9 +22,7 @@ Custom server keystore can be supplied with "-DtlsKeystore=path/to/file.jks".
 
 Custom server keystore password can be supplied with "-DtlsKeystorePassword=MyCustomPassword".
 
-Debug output
-
-See src/main/resources for the log4j2.xml
+Logging level output can be managed with "-Dloglevel=INFO".
```

Comment:
Link to possible options?

Reply:
Added a link

Reply:
Added a clarification to mention it only affects the internal component logging and not the severity of the event itself